SparkPost/2web2ui
# FE-1281 - Remove use of `generateCypressEnv` script when running integration tests. Remove end-to-end tests.

[FE-1281](https://sparkpost.atlassian.net/browse/FE-1281)

### What Changed

- Removes end-to-end tests, adds placeholder `.gitkeep` file to preserve the directory for future use
- Removed contents of all `test-e2e` scripts - will be used in the future
- Removed references to `Cypress.env('USERNAME')` and `Cypress.env('PASSWORD')` - replaced with global constants that can (eventually) reference environmental variables as needed.

### How To Test

- Verify Cypress tests continue to pass
- Verify no end-to-end tests remain in the repo
- Verify the `generateCypressEnv` script was deleted

### To Do
